### PR TITLE
[161417464] Update on card design and event handling, coverage of a major area of…

### PR DESCRIPTION
**Pull request by linsock**
… interaction without covering action items

Looking at the current design of the card and tried, without redesign everything, to reach a most-common user interaction areas coverage for the event handling of the mainAction on the card component.
In the screen attached you can see the bounding boxes around the components to highlight the available area to interact with; Are "touchable" the red, yellow and blue one (without interferences with action buttons).
Wait for a feedback on best actions based on user interaction.
Screenshot 2019-04-24 at 15 45 56

### CODE REVIEW

File: ts/components/wallet/card/CardComponent.tsx [hunk 1]
```diff
@@ -260,19 +264,20 @@ export default class CardComponent extends React.Component<Props> {
...
+             
               <Text style={[CreditCardStyles.smallTextStyle]}>
                 {`${HIDDEN_CREDITCARD_NUMBERS}`}
               </Text>
+
               <Text style={[CreditCardStyles.largeTextStyle]}>
                 {`${wallet.creditCard.pan.slice(-4)}`}
               </Text>
+
+              <View style={{borderWidth:3, borderColor:"yellow",width:"35%"}}></View>
```
Comment (fpersico) [inline ts/components/wallet/card/CardComponent.tsx]: No inline style

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cloudify** [comment]
@linsock please fix the errrors

**Comment by cloudify** [comment]
@linsock can you provide a screenshot?

**Comment by linsock** [comment]
Card component after last fix
image

**Pull request merged**